es-space-mgmt/main/default/classes/reservationManagerController.cls

```diff
@@ -6,7 +6,7 @@ public with sharing class reservationManagerController {
 
         List<customerServices.Customer> customers = new List<customerServices.Customer>();
        if(sObjectType == 'Lead'){
-            for(Lead l:[SELECT Id, Email, Name, City, State, Status FROM Lead WHERE Status IN ('Open - Not Contacted') LIMIT 5]){
+            for(Lead l:[SELECT Id, Email, Name, City, State, Status FROM Lead WHERE Status IN ('Open - Not Contacted', 'New - Not Contacted') LIMIT 5]){
```

Comment:
Can we add `, 'New' ` here as well, in case someone chooses a Professional or Enterprise org? Or do we not want this app to be compatible with those editions? @zaynelt @muenzpraeger

Reply:
I added the 'New' value for now.